application/language/english/ip_lang.php

```diff
@@ -332,7 +332,7 @@
     'no_overdue_invoices'                          => 'No overdue Invoices',
     'no_quotes_requiring_approval'                 => 'There are no quotes requiring approval.',
     'no_updates_available'                         => 'No updates available.',
-		'no_update_invoice_due_date_mail'							 => 'Disable the change of invoice date and due date before emailing',
+		'no_update_invoice_due_date_mail'							 => 'Disable the change of invoice due date before emailing',
```

Comment:
Just the indenting, nothing special.
Also please align the arrow with the other arrows

Reply:
Should be fixed